src/stats/src/GaussianJointPdf.C

```diff
@@ -139,17 +139,23 @@ GaussianJointPdf<V,M>::actualValue(
 
   queso_require_equal_to_msg(domainVector.sizeLocal(), this->m_domainSet.vectorSpace().dimLocal(), "invalid input");
 
-  queso_require_msg(!(gradVector || hessianMatrix || hessianEffect), "incomplete code for gradVector, hessianMatrix and hessianEffect calculations");
+  queso_require_msg(!(hessianMatrix || hessianEffect), "incomplete code for gradVector, hessianMatrix and hessianEffect calculations");
 
   double returnValue = 0.;
 
-  if (this->m_domainSet.contains(domainVector) == false) { // prudenci 2011-Oct-04
+  if (this->m_domainSet.contains(domainVector) == false) {
```

Comment:
What is this logical testing for? Is it just testing if there are no model parameters for this pdf? If that is the case, does one want to return 0, instead of an error?